internal/controller/services/monitoring/resources/monitoring-stack.tmpl.yaml

```
@@ -22,4 +22,4 @@ spec:
     requests:
       cpu: {{ .CPURequest }}
       memory: {{ .MemoryRequest }}
-  retention: {{ .StorageRetention }}
+  retention: {{ .StorageRetention }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)